tests/components/homewizard/conftest.py

```diff
@@ -65,6 +72,59 @@ def mock_homewizardenergy(
         yield client
 
 
+@pytest.fixture
+def mock_homewizardenergy_v2(
+    device_fixture: str,
+) -> MagicMock:
+    """Return a mock bridge."""
+    with (
+        patch(
+            "homeassistant.components.homewizard.HomeWizardEnergyV2",
+            autospec=True,
+        ) as homewizard,
+        patch(
+            "homeassistant.components.homewizard.config_flow.HomeWizardEnergyV2",
+            new=homewizard,
+        ),
+    ):
+        client = homewizard.return_value
+
+        client.combined.return_value = CombinedModels(
+            device=Device.from_dict(
+                load_json_object_fixture(f"v2/{device_fixture}/device.json", DOMAIN)
+            ),
+            measurement=Measurement.from_dict(
+                load_json_object_fixture(
+                    f"v2/{device_fixture}/measurement.json", DOMAIN
+                )
+            ),
+            state=(
+                State.from_dict(
+                    load_json_object_fixture(f"v2/{device_fixture}/state.json", DOMAIN)
+                )
+                if get_fixture_path(f"v2/{device_fixture}/state.json", DOMAIN).exists()
+                else None
+            ),
+            system=(
+                System.from_dict(
+                    load_json_object_fixture(f"v2/{device_fixture}/system.json", DOMAIN)
+                )
+                if get_fixture_path(f"v2/{device_fixture}/system.json", DOMAIN).exists()
+                else None
+            ),
+        )
+
+        # device() call is used during configuration flow
+        client.device.return_value = client.combined.return_value.device
+
+        # Authorization flow is used during configuration flow
+        client.get_token.return_value = Token.from_dict(
+            load_json_object_fixture("v2/generic/post_user_200_ok.json", DOMAIN)
```

Comment:
is the rest of the generic ones used? I would just rename this to token.json or something along the lines

Reply:
Oh, I forgot to add unhappy-flow tests for those. Good catch.